### PR TITLE
toast component

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: components/alert.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+      <div class="alert success">
+          <strong>Success!</strong> This is a Success Alert
+      </div>
+
+      <div class="alert warning">
+          <strong>Warning!</strong> This is a Warning Alert   
+          
+      </div>
+      <div> <img class='code-img' src="../assets/alert.png"/></div>
```
Comment (Naveenchandar) [inline components/alert.html:81]: Please provide alt description for accessibility.

File: components/alert.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+          
+      </section>
+     
+       
+
```
Comment (Naveenchandar) [inline components/alert.html:87]: You can remove the extra space

File: components/badge.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+         </div>
+         <div><img class='code-img' src="../assets/badge.png"/></div>
+    
+
+
+      
+            
+                
+            
+
```
Comment (Naveenchandar) [inline components/badge.html:110]: You can remove the extra space

File: main.css [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+    gap: 13rem;
+    background-color: antiquewhite;
+     }
+
+     .main{
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 2rem;
+      }
```
Comment (Naveenchandar) [inline main.css:101]: code alignment missing. To align and format the code, you can use prettier extension available in vs code or you can use some other tools available in online.

**Comment by Rohitprasad83** [comment]
please add a screenshot, gif or a preview link of the component.

File: main.css [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+    border-radius: 50%;
+    width: 100%;
+    height: inherit;
+    object-fit: cover; 
+    width: 14rem; 
+  }
+  .badge-online{
+    border-radius: 50%;
+    position: absolute;
+    display: flex;
```
Comment (Naveenchandar) [inline main.css:118]: If flex property is repeated often, you can either declare a variable or separate class and utilise it.

File: main.css [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+}
+  
+.rounded-img {
+    margin: 2rem;
+    border-radius: 50%;
+    width: 30rem;
+    height: inherit;
+    object-fit: cover;
+}
+toast-heading{
```
Comment (Naveenchandar) [inline main.css:266]: Invalid selector. Either use dot if it's a class name or # if it's id.

**Pull request merged**